SConstruct

```diff
@@ -32,29 +51,60 @@ if env["mode"] == "profile":
 # Required build flags. If you want to use SSE optimization, you can turn on
 # -msse3 or (if just building for your own computer) -march=native.
 env.Append(CCFLAGS = flags)
-env.Append(LIBS = [
-	"SDL2",
-	"png",
-	"jpeg",
-	"GL",
-	"GLEW",
-	"openal",
-	"pthread"
-]);
-# libmad is not in the Steam runtime, so link it statically:
-if 'SCHROOT_CHROOT_NAME' in os.environ and 'steamrt_scout_i386' in os.environ['SCHROOT_CHROOT_NAME']:
-	env.Append(LIBS = File("/usr/lib/i386-linux-gnu/libmad.a"))
-elif 'SCHROOT_CHROOT_NAME' in os.environ and 'steamrt_scout_amd64' in os.environ['SCHROOT_CHROOT_NAME']:
-	env.Append(LIBS = File("/usr/lib/x86_64-linux-gnu/libmad.a"))
+if msys:
+	env.Append(LIBS = [
+		"winmm", # for timeBeginPeriod
+		"mingw32",
+		"SDL2main",
+		"SDL2.dll",
+		"opengl32.dll",
+		"glew32.dll",
+		"png.dll",
+		"jpeg.dll",
+		"mad.dll",
+		"openal.dll"
+	])
+	env.Append(LINKFLAGS = ["-Wl,--subsystem,windows"])
```

Comment:
Spaces after comma operators

Reply:
That's weird, there is no space after the comma in mine.

Reply:
If you mean LINKFLAGS, then it's supposed to be like that. Thats only one option.

Reply:
Oops, didn't see the opening quotes.